ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_mathematical_functions.py

```diff
@@ -2819,6 +2819,40 @@ def test_jax_remainder(
     )
 
 
+@handle_frontend_test(
+    fn_tree="jax.numpy.roots",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=1,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=0,
+    ),
+)
+def test_jax_roots(
+    *,
+    dtype_and_x,
+    test_flags,
+    on_device,
+    fn_tree,
+    frontend,
+    backend_fw,
+):
+    input_dtype, x = dtype_and_x
+    assume("float16" not in input_dtype)
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        frontend=frontend,
+        backend_to_test=backend_fw,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        p=x,
```

Comment:
I believe your issue will be solved with this
```suggestion
        p=x[0],
```